packages/react-day-picker/src/hooks/useDayModifiers/useDayModifiers.ts

```diff
@@ -0,0 +1,51 @@
+import { ModifierStatus } from '../../types/Modifiers';
+import { useDayPicker } from '../../contexts/DayPicker';
+import { useModifiers } from '../../contexts/Modifiers';
+import { getModifierStatus } from './utils/getModifierStatus';
+
+export type DayModifiers = {
+  /** The status of the modifiers */
+  modifiers: ModifierStatus;
+  /** The class names based on modifiers. */
+  modifierClassNames: string[];
+  /** The style based on modifiers. */
+  modifierStyle: React.CSSProperties;
+};
+
+/** Return the modifiers and its styles for the specified date. */
+export function useDayModifiers(date: Date): DayModifiers {
+  const context = useDayPicker();
+
+  // TODO: Rename this hook useModifiersContext?
```

Comment:
I am wondering too – if we should change the others too. Let's keep like this ATM

```suggestion
```